packages/app-check/ios/RNFBAppCheck/RNFBAppCheckProviderFactory.m

```diff
@@ -16,10 +16,7 @@
  */
 
 #import "RNFBAppCheckProviderFactory.h"
```

Comment:
this one imports RNFBAppCheckProvider.h, which imports the others, including them again below caused the loop which caused the warning